### ISSUE TITLE
Beach names missing

### ISSUE THREAD

**Issue by laem** [openmaptiles/openmaptiles#1609]
Hi, I'm missing beach names. 

Check for instance this place in Bretagne, France. 

![image](https://github.com/openmaptiles/openmaptiles/assets/1177762/7c5147ee-5767-4de7-b4a3-ecea3c007c67)

It looks reasonable to me that beach names are as much important as neighborhood names, for instance. Often, they are one of the main attractions of a city. They are well visible on the map, but there are often multiple beach names for every coastal city. Also sometimes, beaches don't map to cities : they can be a lot more known than the surrounding cities themselves. 

In the map above, the "La Fosse" label is of arguably of interest for very local searches, while the beach just above (called Longchamps) might be on par with the city itself (Saint-Lunaire) in this coastal conurbation. As you can see, the beach spans two cities : Saint-Lunaire and Saint-Briac.

**TLDR** : beach names are, I think, as important as park names, but I can't find them in the OpenMapTiles spec :)

**Comment by laem** [comment]
Solution : it doesn't look hard to add a beach label. But beaches are in landcover. I'm not familiar enough to propose a change in the yaml config files.

**Comment by TomPohys** [comment]
Thanks @laem for pointing this out. The `landcover` layers do not contain `name` attribute. Will be better to extend the `poi` layer for `natural=beach` and keep the points with the name of the beach there. 

PR is welcome.
